cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/type/HostCommissionState.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.cloudbreak.common.type;
+
+public enum HostCommissionState {
+
+    COMMISSIONED("COMMISSIONED"),
+
+    DECOMMISSIONED("DECOMMISSIONED"),
+
+    UNKNOWN("UNKNOWN"),
+
+    OFFLINING("OFFLINING"),
+
+    OFFLINED("OFFLINED");
```

Comment:
Are these the states mapped from the CM result for the same field?

Reply:
Yes these are mapped to the ApiCommissionState which is returned by CM

Reply:
what happens when we meet with older CM?

Reply:
Was this resolved ? Seems like a very valid concern. Maybe these specific fields have existed forever - so not a concern?